_posts/2017-12-21-14-01-netcore-beta1.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title : "14.1 7.x Beta1"
+description: "7.x Beta1 for ASP.NET and ASP.NET Core"
+category: "14. 7.x Features "
+---
+
+**Web API OData Beta1** includes a new package for [OData V7.0.0 for ASP.NET Core 2.x](https://www.nuget.org/packages/Microsoft.AspNetCore.OData/). 
+A nightly version of this package is available using this url: [https://www.myget.org/F/webapinetcore/api/v3/index.json](https://www.myget.org/F/webapinetcore/api/v3/index.json).
+
+There is also a new Web API OData V7.0.0 for ASP.NET available as an Alpha, which you can access by downloading the packages
+using this url: [https://www.myget.org/F/webapinetcore/api/v3/index.json](https://www.myget.org/F/webapinetcore/api/v3/index.json).
+
+Both packages depends on [OData Lib 7.0.0](https://www.nuget.org/packages/Microsoft.OData.Core/7.0.0).
+
+The code for the packages can be found here:
+[https://github.com/OData/WebApi/tree/feature/netcore](https://github.com/OData/WebApi/tree/feature/netcore)
+
+### Known Issues
+
+**Web API OData for ASP.NET Core**, has following limitations which are known issues:
+* Batching is not supported
+* Using EnableQuery in an HTTP route, i.e. non-ODta route, is not fully functional
+
+**Web API OData for ASP.NET** , there are no known issues.
+
+### OData V7.0.0 for ASP.NET Core 2.x
+
+The new OData V7.0.0 for ASP.NET Core 2.x package supports the same features set as Web API OData V6.0.0 but works with ASP.NET Core.
+You can learn more about ASP.NET Core from the [documentation](https://docs.microsoft.com/en-us/aspnet/core/).
+
+To get started with OData V7.0.0 for ASP.NET Core 2.x, you can use code that is very similar to Web API OData V6.0.0. All of the
+documentation in [Writing a simple OData V4 service](http://odata.github.io/WebApi/#01-02-getting-started) is correct except for
+configuring the OData endpoint. Instead of using the Register() method, you'll follow the new service + route configuration model
+used in ASP.NET Core.
+
+The namespace for both Web API OData packages is Microsoft.AspNet.OData.
+
+Let's get started by creating a simple OData V4 service. It has one entity set `Products`, one entity type `Product`. `Product`
```

Comment:
Maybe use some codes is better.

Reply:
I'll delete this since the code is provided below.